From 2d73469c7a17ebfe4330ac6643b0c8abdc125d05 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 30 Jan 2019 09:29:44 -0800
Subject: [PATCH] get-pac-test: Fix build with clang/libc++

get-pac-test.cpp:55:10: error: assigning to 'int' from incompatible type '__bind<int &, sockaddr *, unsigned int>'
                        ret = bind(m_sock, (sockaddr*)&addr, sizeof (struct sockaddr_in));                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Upstream-Status: Submitted [https://github.com/libproxy/libproxy/pull/97]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 libproxy/test/get-pac-test.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libproxy/test/get-pac-test.cpp b/libproxy/test/get-pac-test.cpp
index 0059dfb..911f296 100644
--- a/libproxy/test/get-pac-test.cpp
+++ b/libproxy/test/get-pac-test.cpp
@@ -52,7 +52,7 @@ class TestServer {
 
 			setsockopt(m_sock, SOL_SOCKET, SO_REUSEADDR, &i, sizeof(i));
 
-			ret = bind(m_sock, (sockaddr*)&addr, sizeof (struct sockaddr_in));
+			ret = ::bind(m_sock, (sockaddr*)&addr, sizeof (struct sockaddr_in));
 			assert(!ret);
 
 			ret = listen(m_sock, 1);
-- 
2.20.1

